subnetGroups/subnet-groups.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "subnetGroupName": "Defi Kingdoms",
+    "explorerUri": "defi-kingdoms",
```

Comment:
Since these values aren't uris, should we clarify in the name: `explorerUriId` or something like that.

Reply:
Ah, I see where the confusion came from.  I suppose that these actually are Uris then.